---
title: "Projects"
format:
  html:
    theme: brite
    css: styles.css
    toc: true
---

## Research directions

::: {.projects-grid}

::: {.project-card}
### Designing functional proteins

The engineering of proteins with enhanced native functions or entirely novel activities are major challenges in computational protein design. However, current generative models exhibit limitations in producing functionally diverse variants. Thus, new methods are needed that can (i) extract the right patterns from evolutionary data, (ii) incorporate structural constraints, and (iii) leverage experimental data to enrich the functional generative landscape. My doctoral research focuses on developing computational approaches towards these goals.

**Collaborators:** [Thomas Hopf](https://thomashopfconsulting.com){.underline}, [Marks Lab](https://marks.hms.harvard.edu/), [Sander Lab](https://sanderlab.org/).

**Status:** *In Progress*

:::

::: {.project-card}
### EVEdesign

[EVEdesign](https://evedesign.bio){.underline}, led by [Thomas Hopf](https://thomashopfconsulting.com){.underline}, is an open-source initiative to democratise AI-driven protein design. It enables end-to-end sequence design, from entering the target protein through analysing the generated library to ordering codon-optimised DNA sequences. EVEdesign also aims to be a flexible, model-agnostic tool, where both single mutation scans and higher-order mutant generation are possible with custom sampling techniques. Mutation sites are readily mapped onto available structures, and mutant information is extracted and visualised in sequence space, all within the same analysis window. I am working on the creation of our first complex protein design pipeline, which will be included in the pre-print.

**Collaborators:** [Thomas Hopf](https://thomashopfconsulting.com){.underline}, [Marks Lab](https://marks.hms.harvard.edu/), [Steinegger Lab](https://steineggerlab.com/), [D'Oelnitz Lab](https://simondoelsnitz.com), [Sander Lab](https://sanderlab.org/), [NIST](https://www.nist.gov).

**Status:** *In Progress*

**Links:**  [EVEdesign](https://evedesign.bio){.underline}
:::
:::

## Other initiatives, collaborations and projects

::: {.projects-grid}

::: {.project-card}
### Sanger Machine Learning Bookclub

I co-founded and currently lead the [Sanger Machine Learning Boocklub](https://brazen-sherbet-4ef.notion.site/ML-bookclub-Sanger-20e711a8a64c80eda9e6c52788a7f5cb){.underline} (SMLB), a community of nearly 100 people that brings together "BioML" enthusiasts across the Wellcome Sanger Institute and the European Bioinformatics Institute (EMBL-EBI). At the SMLB we aim to focus on the theory of Machine Learning methods that are advancing biological and biotechnological research. It was created to share knowledge, resources and research in an engaging way, and to bring curious scientists together. We run weekly sessions in one of the following forms: workshops, study sessions, Journal Club-style sessions and research talks. A record of all sessions is kept at the official [SMLB Notion site](https://brazen-sherbet-4ef.notion.site/ML-bookclub-Sanger-20e711a8a64c80eda9e6c52788a7f5cb){.underline}.

**Dates** *04/2025 - Present*

**Links:** [SMLB site](https://brazen-sherbet-4ef.notion.site/ML-bookclub-Sanger-20e711a8a64c80eda9e6c52788a7f5cb){.underline}
:::

::: {.project-card}
### Information-theoretical tools for single-cell RNA-seq data analysis

Elucidating structure from single-cell RNA-seq data is a major challenge in modern biological research. For my final project of the MRes in Bioinformatics and Theoretical Systems Biology, I improved existing and developed novel information-theoretical tools to uncover real relationships between genes as part of the single-cell RNA-seq data analysis pipeline. This work introduced theoretically tractable alternatives to available methods. Further developments from the [Theoretical Systems Biology group](https://www.perez-carrasco.com/){.underline} and part of my work are now being prepared for publication.

**Collaborators:** [Theoretical Systems Biology group](https://www.perez-carrasco.com/){.underline}, [Briscoe Lab](https://www.crick.ac.uk/research/labs/james-briscoe){.underline}

:::

::: {.project-card}
### Predicting DNA function when grafted across species

For one of my rotations in the PhD programme, I was part of a larger genome engineering project led by the [Parts Lab](https://www.sanger.ac.uk/group/parts-group/){.underline} focused on inter-species chromosome transplantation. I developed an approach that involved fine-tuning DNA language models to elucidate the extent to which the effects of chromosome grafting could be learned. I summarised my findings in a [poster](poster_final.pdf){.underline} which earned the Best Poster Award within the PhD cohort. Manuscript in preparation.

**Collaborators:** [Parts Lab](https://www.sanger.ac.uk/group/parts-group/){.underline}

:::

::: {.project-card}
### AI-readiness standards for space biology

I am part of an international intiative led by scientists from the European Space Agency (ESA) and the National Aeronautics and Space Administration (NASA) that aims to introduce AI-readiness standards and guidelines for space biology data, with ramifications for other biological data types. Manuscript in preparation.

**Collaborators:** [NASA](https://www.nasa.gov){.underline}, [ESA](https://www.esa.int){.underline}

:::

:::
